### PR TITLE
Infomodal

### PR DESCRIPTION
**Pull request by MattWaraich**
# Drawer Menu
- Add drawer right 
- Add navigation routes 
- Style drawer menu

### CODE REVIEW

File: client/ios/Podfile.lock [hunk 1]
```diff
@@ -357,6 +357,6 @@ SPEC CHECKSUMS:
   RNScreens: f28b48b8345f2f5f39ed6195518291515032a788
```
Comment (TienSFU25) [inline client/ios/Podfile.lock:369]: why is this file changed? are you sure this should be in the commit?

File: client/js/navigation/DrawerScreen.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+            }}>
+            About Us
+          </Text>
+        </TouchableOpacity>
+        <TouchableOpacity
+          onPress={() => this.navigateToScreen('Contact')}
+          style={{borderBottomColor: 'white', borderBottomWidth: 1}}>
+          <Text
+            style={{
+              fontSize: 20,
```
Comment (TienSFU25) [inline client/js/navigation/DrawerScreen.js:41]: there's duplicate styling in this file (fontsize: 20, padding 25, color white), can you put this in Stylesheet.create?

File: client/ios/Podfile.lock (hunk 1 above)
Comment (MattWaraich) [reply client/ios/Podfile.lock:369]: Yes.

**Review by TienSFU25 — DISMISSED** [review]

File: yarn.lock [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
```
Comment (siddharthparmar7) [inline yarn.lock]: Please delete this file

Comment (MattWaraich) [reply yarn.lock]: complete.

File: client/js/components/FaqForum/FaqForum.js [hunk 4]
```diff
@@ -66,16 +62,27 @@ class FaqForum extends React.Component {
...
             <Animated.Text
               style={{
-                color: '#9963ea',
+                color: '#000000',
                 fontSize: 20,
+                paddingRight: 10,
                 transform: [{rotate: spin}],
               }}>
-              {icon}
+              {this.state.collapse ? (
```
Comment (ghsyeung) [inline client/js/components/FaqForum/FaqForum.js]: This could simplify to 

```
<Image source={
    this.state.collapse ? require(this.icon.collapse) : require(this.icon.show)
 }/>
```

**Review by ghsyeung — COMMENTED** [review]
looking good, just minor fixes

File: client/js/components/FaqForum/FaqForum.js (hunk 4 above)
Comment (MattWaraich) [reply client/js/components/FaqForum/FaqForum.js]: Completed.

**Review by siddharthparmar7 — APPROVED** [review]
